### PR TITLE
Adding MIT license

### PR DESCRIPTION
**Pull request by yaromins**
Would be nice to have some license on file to know what is acceptable to do with the code

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (ilkome) [inline LICENSE]: The MIT License (MIT)

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Copyright (c) 2021  Ilya Komichev
```
Comment (ilkome) [inline LICENSE]: Copyright (c) 2018-present, Ilya Komichev

**Review by ilkome — CHANGES_REQUESTED** [review]

**Comment by yaromins** [comment]
@ilkome, requested changes are done

**Comment by ilkome** [comment]
Thanks :)

**Pull request merged**